src/Durable/SetFunctionInvocationContext.cs

```diff
@@ -17,16 +17,16 @@ namespace Microsoft.Azure.Functions.PowerShellWorker.Durable
     public class SetFunctionInvocationContextCommand : PSCmdlet
     {
         internal const string ContextKey = "OrchestrationContext";
-        private const string StarterKey = "OrchestrationStarter";
+        private const string OrchestrationClientKey = "OrchestrationClient";
 
         [Parameter(Mandatory = true, ParameterSetName = ContextKey)]
         public OrchestrationContext OrchestrationContext { get; set; }
 
         /// <summary>
-        /// The orchestration client output binding name.
+        /// The orchestration client.
         /// </summary>
-        [Parameter(Mandatory = true, ParameterSetName = StarterKey)]
-        public string OrchestrationStarter { get; set; }
+        [Parameter(Mandatory = true, ParameterSetName = OrchestrationClientKey)]
+        public object OrchestrationClient { get; set; }
 
         [Parameter(Mandatory = true, ParameterSetName = "Clear")]

```

Comment:
Sorry I missed this on my previous review. Why is this switch mandatory?

Reply:
No idea, fixing.

Reply:
Oh, no, I understand now. This is because this switch is used for selecting the parameter set: for this parameter set ("Clear"), this switch is always mandatory because this is what makes PowerShell select this parameter set. If this switch is not provided, a different parameter set will be used. No fix required. But good catch. :-)

Reply:
Got it. Thanks for clarifying.